tests/dev/script-test.R

```diff
@@ -0,0 +1,24 @@
+# Test Script for workflow
+
+rm(list=ls())
+
+library(tlf)
+library(ospsuite)
+library(ospsuite.reportingengine)
+
+#sim <- loadSimulation("../tests/data/S1.pkml")
+#pop <- loadPopulation("../tests/data/pop_10.csv")
+
+popWorkflow <- PopulationWorkflow$new(outputFolder = "../tests/WorkflowOutput",
+                                      simulationPath = "../tests/data/S1.pkml",
+                                      populationPath = "../tests/data/pop_10.csv"
+                                      )
+
+popWorkflow$runWorkflow()

```

Comment:
We have to think about the api here. How do we turn on and off part of the workflow
If the user does not see what actions are part of the workflow, how does he know what to enable and disable?

Reply:
That was the reason I defined the Tasks with:
- a logical field called active (or I can relabel isActive) and methods called $activate() and $inactivate() to set the value as TRUE/FALSE
- a print() method showing if the name of the task and whether is active for the workflow 
e.g. from the example above: 
> popWorkflow$demographyPlot
[1] "Task: Plot demography"
[1] "Task to be performed in workflow: TRUE"
[1] "Task already performed in workflow: TRUE"

I was planning to create a similar print() method for the PopulationWorkflow class which would directly print a data.frame with task name, active, available output (so you don't need to re-run existing output):
> popWorkflow
|task                      |active |available.ouptut |
|:-------------------------|:------|:----------------|
|Simulate population       |TRUE   |TRUE             |
|Calculate PK parameters   |TRUE   |FALSE            |
|Analyze sensitivity       |TRUE   |FALSE            |
|Plot demography           |TRUE   |TRUE             |
|Plot time profile         |TRUE   |TRUE             |
|Plot PK parameters        |TRUE   |FALSE            |
|Plot sensitivity analysis |TRUE   |FALSE            |

Reply:
neat. Let's check it out tomorrow